transform/transformers/PCKTransformer.py

```diff
@@ -40,7 +47,19 @@ def get_form_questions(self):
     def get_cs_form_id(self):
         instrument_id = self.response['collection']['instrument_id']
 
-        return self.form_ids[instrument_id]
+        try:
+            form_type = self.form_types[self.survey['survey_id']]
+        except KeyError:
+            logger.error("Invalid survey id '{}'".format(self.survey['survey_id']))
+            return 'XXXXXX'
```

Comment:
This will be changed to return `None` on a `KeyError`. @tundish - This will impact the rendering of the templates where `form_id` is used.